src/applications/disability-benefits/new-686/config/form.js

```diff
@@ -0,0 +1,56 @@
+// import fullSchema from 'vets-json-schema/dist/21-686-schema.json';
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+// const { } = fullSchema.properties;
+
+// const { } = fullSchema.definitions;
```

Comment:
If this is unnecessary, should it be deleted instead of commented out?

Reply:
These come with the form generator, again this branch is just for us to test out building this form from scratch and is NOT a final version.